modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -239,7 +239,7 @@ private void _validate(ObjectEntry objectEntry) {
 	}
 
 	private static final Pattern _pathPattern = Pattern.compile(
-		"^[/][a-zA-Z0-9][a-zA-Z0-9-/]{1,255}");
+		"/[a-zA-Z0-9][a-zA-Z0-9-/]{1,255}");
```

Comment:
By the way, I think we modified this expression in a wrong way. The purpose is to have a maximum of 255 characters, but we are allowing 256 "/" + 255 extra

```
commit 28af07aa7c3e3f6a7d58e3b1f56b1067b655e2af
Author: sergiojimcos <sergio.jdelcoso@liferay.com>
Date:   Mon Jul 17 16:27:32 2023 +0200

    LPS-190723 Update regular expression where an endpoint should start with a SLASH character

diff --git a/modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java b/modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java
index 08413787e464..07fefdf7127f 100644
--- a/modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java
+++ b/modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java
@@ -223,7 +223,7 @@ public class APIEndpointRelevantObjectEntryModelListener
        }
 
        private static final Pattern _pathPattern = Pattern.compile(
-               "[a-zA-Z0-9-/]{1,255}");
+               "^[/][a-zA-Z0-9][a-zA-Z0-9-/]{1,255}");

```

Please @jalber786 could you modify it?

cc/ @sergiojimcos

Reply:
You are right @ccorreagg! Thanks for catching that

Reply:
I updated the comment as we are even allowing 257, not 256

Reply:
What if it ends with a slash? Are we ok with that?

Reply:
good point, I think that it should be validated also, in order to no end with a slash